rubix-rpm/src/main/resources/rpm/scripts/configure-presto.sh

```diff
@@ -2,8 +2,6 @@
 
 RUBIX_HOME_DIR=/usr/lib/rubix
 RUBIX_CACHE_DIR=/var/lib/rubix/cache
-PRESTO_HOME_DIR=/usr/lib/presto
-HADOOP_HOME_DIR=/usr/lib/hadoop
 
 # Configure BookKeeper server for PrestoClusterManager
 RUBIX_SITE=/etc/rubix/rubix-site.xml
```

Comment:
Instead of `/etc/rubix/rubix-site.xml`, can we use the `/usr/lib/rubix/etc/rubix-site.xml` as the default `rubix-site.xml` location?